README.markdown

```diff
@@ -187,7 +187,15 @@ let colour = "red"
 
 ## Code Organization
 
-Use extensions to organize your code into logical blocks of functionality. Each extension should be set off with a `// MARK: -` comment to keep things well-organized.
+Use extensions to organize your code into logical blocks of functionality. Each extension should be set off with a the following comment to keep things well-organized.
+
+```
+    //==========================================================================
+    // MARK: - Something
+    //==========================================================================
+```
```

Comment:
Are we still using this kind of Marks?

I am not a super fan of it (specially now that we get to see "normal" marks in xcode minimap.

Reply:
We are still using these right now. I've been using them since Swift started because prior to that I used pragma marks in Objective C. Pragma marks had the benefit of being a different color from other comments. Thats why I added the extra lines above and below just to make scanning for sections easier during a fast scroll. But maybe they aren't necessary anymore with all the Xcode features that have been built around marks.

Reply:
Let's see if the rest of the team ❤️ or 💔 them

Reply:
Marks show in the minimal but you have to use the minimap to get that value 😛. I like this style for the same reason as Ryan. If I'm scrolling through a file I will miss `// Mark: -` every time

Reply:
One note for, one note against.

For: We have an Xcode snipping so you just type `//mark` and it will prefill this entire style and put your cursor on the title. It's very streamlined. So don't worry about pain of making them, just their value or detriment for visibility.

Against: Xcode 11 now has a color option for marks. So you can make marks red while the rest of the comments stay gray. It only works for the title of the mark unfortunately so its minimal visibility, but its something.

Reply:
Having a different color helps but it doesn't replace having a physical line where a section starts/stops, which is the main thing that lets these marks stand out

Reply:
You also have a physical mark now in Xcode (not sure if you need to turn it on..)
<img width="1454" alt="Screen Shot 2020-08-28 at 7 10 36 AM" src="https://user-images.githubusercontent.com/994334/91577522-acb2f280-e8fd-11ea-9613-37c66f970849.png">

Reply:
Oh I know that line is there, but it's barely there

Reply:
I'm with Peter. The current styling of the Mark really stands out and I'm less likely to miss it or scroll past.

Reply:
I see your point. 
Maybe I don't like it because I see and use marks in a different way.. 

I don't use marks while reading code. Right now, marks stand out way too much for me, they are more important than code itself.

I want marks to be there when I need them. I think they are very useful when using xcode's jump bar (image attached), as they provide great context. The jump bar is also great when you land on a new file and want to see its skeleton/interface. But while reading code, I am still not a fan of them being so prominent.

<img width="312" alt="Screen Shot 2020-08-28 at 7 28 33 AM" src="https://user-images.githubusercontent.com/994334/91580370-d837dc00-e901-11ea-8629-03dc15deedb1.png">

Reply:
I could learn to live without the large mark, but if other people also find it helpful then it seems worth keeping

Reply:
I agree if you guys find it useful I will tune my theme to slightly dim the comments 😛

Reply:
I don't mind the large MARKS. 

I like the jump bar, especially for `CTRL+6`ing when navigating extra long files, but it irks me that you can't just keep the jump bar contents open permanently in a panel. (or can you? 👀) So I usually rely on code folding keyboard commands to get the overview of a file.

I also don't find myself utilizing the mini map very often. It looks cool tho.

That said, if we remove the large MARKS, I might lean on the jump bar / mini map more often.

Reply:
Let's keep them for now then. I have already added the snippet 😛